AndrewTehan/hometask-4/my_hash.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# this class is my realisation of Hash
+class MyHash
+  include Enumerable
+
+  def initialize
+    @keys_values = []
```

Comment:
`keys_values` seems not really extensible. It's not a very good idea to describe everything that var contains (or method does) in naming.

Probably `dictionary` suits here better, what do you think?